src/platforms/android/configuration/integrations/jetpack-compose.mdx

```diff
@@ -8,11 +8,49 @@ categories:
   - mobile
 ---
 
-The `sentry-compose` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) Navigation support for Sentry using the [SentryNavigationIntegration](https://github.com/getsentry/sentry-java/blob/524ee49b212c3f2eead20960c3c6825ccdbf8007/sentry-compose/src/androidMain/kotlin/io/sentry/compose/SentryNavigationIntegration.kt). The source can be found [on GitHub](https://github.com/getsentry/sentry-java/blob/main/sentry-compose/src/androidMain/kotlin/io/sentry/compose/).
+The `sentry-compose` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) Navigation support for Sentry.
 
 On this page, we get you up and running with Sentry's Compose Navigation Integration, so that it will automatically add a breadcrumb and start a transaction for each navigation event.
 
-## Install
+## Auto-Installation With the Sentry Android Gradle Plugin
+
+If you're using the [Sentry Android Gradle Plugin](/platforms/android/gradle/), the Jetpack Compose Navigation Integration is automatically applied. In this case both breadcrumbs and transactions are automatically created for every navigation event.
```

Comment:
Can we still add the `### Install` header here? For example, this is how the structure looks for OkHttp docs:
![image](https://user-images.githubusercontent.com/4999776/198664186-3872d97f-d5f4-4945-a4cc-3a3a78bbc0ec.png)

So I'd like to align them for consistency.

Reply:
Also, the code snippet for using the Gradle plugin is missing - so just check the okhttp page maybe for prior art. Thanks ;)